src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -183,6 +184,22 @@ bool mongoConnect(const char* host,
 
     LM_I(("Successful connection to database"));
 
+    //
+    // WriteConcern
+    //
+    enum WriteConcern wconcern = (writeConcern == "normal")? W_NORMAL : W_NONE;
```

Comment:
Write concerns is, from a semantic point of view, an integer number. Have a look to http://docs.mongodb.org/manual/core/write-concern/ and, if it not clear after then, let's talk by skype.